### ISSUE TITLE
Greybird xfwm4 theme for HiDPI (Retina, 192 dpi)

### ISSUE THREAD

**Issue by Yanpas** [shimmerproject/Greybird#100]
Currently hidpi displays are not supported by xfwm4 greybird theme

**Comment by ochosi** [comment]
I don't have a 192DPI display so my motivation to tackle this item is low. Furthermore, Gtk2 doesn't really seem to play too nicely with HiDPI yet, so I guess this'll have to wait until Xfce is Gtk3 and we have proper support.

**Comment by noleti** [comment]
Do you have .svg sources for the images? In that case, someone (e.g., me) could probably easily create a high DPI version of this theme. I've been using greybird for a long time, and I would love to continue to use it on my new laptop.

**Comment by ochosi** [comment]
Nope, xfwm4 doesn't support svg and originally everything was done by pushing pixels.

That's the main reason I haven't created the HiDPI version myself yet btw.

**Comment by ochosi** [comment]
There's a usable branch for this now. It's not finished, but min|max|close are there and the basics work: https://github.com/shimmerproject/Greybird/tree/xfwm4-hidpi

**Comment by jnrbsn** [comment]
@ochosi Is this just low priority or is it no longer desired for some reason? If the former, what needs to be done to finish it?
